yt/frontends/enzo/data_structures.py

```diff
@@ -737,7 +741,7 @@ def _parse_parameter_file(self):
         dictionaries.
         """
         # Let's read the file
-        with open(self.parameter_filename, "r") as f:
+        with io.open(self.parameter_filename, "r") as f:
```

Comment:
Why using `io.open`? I thought `io.open == open`.

Reply:
This was to deal with a problem with `libconf.load` in Python 2. Apparently, `open` gives back bytes in Python 2 where `libconf.load` expects unicode. The comments of [this issue](https://github.com/Grk0/python-libconf/issues/4) is where I found this solution.